js/views/pauseScreen.js

```diff
@@ -309,7 +311,7 @@ var PauseScreen = React.createClass({
           <ControlBar
             {...this.props}
             controlBarVisible={this.state.controlBarVisible}
-            playerState={this.state.playerState}
+            playerState={this.props.playerState}
```

Comment:
Is this correctly using props instead of state?

Reply:
Yep, that is how it is passed in other views, as a prop. Considering all views are created in skin.js and are created in the same way, and our new unit test framework was complaining about this, I made this change.